lib/workers/repository/reconfigure/reconfigure-cache.ts

```diff
@@ -0,0 +1,30 @@
+import { logger } from '../../../logger';
+import { getCache } from '../../../util/cache/repository';
+
+export function setReconfigureBranchCache(
+  reconfigureBranchSha: string,
+  configFileName: string,
+  isConfigValid: boolean
+): void {
+  const cache = getCache();
+  const reconfigureBranchCache = {
+    reconfigureBranchSha,
+    configFileName,
+    isConfigValid,
+  };
+  if (cache.reconfigureBranchCache) {
+    logger.debug({ reconfigureBranchCache }, 'Update Reconfigure Branch Cache');
```

Comment:
```suggestion
    logger.debug({ reconfigureBranchCache }, 'Update reconfigure branch cache');
```

Capitalizing all the words is a bit noisy.